browserstack.yml

```diff
@@ -28,27 +29,16 @@ projectName: BrowserStack Samples
 platforms:
   - os: OS X
     osVersion: Big Sur
-    browser: Chrome
+    browserName: Chrome
     browserVersion: latest
   - os: Windows
     osVersion: 10
-    browser: Edge
+    browserName: Edge
     browserVersion: latest
-  - device: Samsung Galaxy S22 Ultra
+  - deviceName: Samsung Galaxy S22 Ultra
     browserName: chrome # Try 'samsung' for Samsung browser
     osVersion: 12.0
 
-# =======================
-# Parallels per Platform
```

Comment:
`parallelsPerPlatform` isn't supported in Robot?

Reply:
will add, missed in update

Reply:
added